spring-boot-project/spring-boot-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -662,7 +662,7 @@ that you have added. Since `spring-boot-starter-web` added Tomcat and Spring MVC
 auto-configuration assumes that you are developing a web application and sets up Spring
 accordingly.
 
-.Starters and Auto-Configuration
```

Comment:
I am not sure that we should change that. Titles in the doc are consistently capitalized. Thoughts?

Reply:
@snicoll Actually I tried to retain the capitalisation there as intended. AFAIK English capitalisation is only for the first letter but I might be wrong as I'm not a native English speaker 😅

Reply:
You are correct. This is also consistent with [this section in the doc](https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#using-boot-auto-configuration). Thanks for the feedback.